File: Byte&Buy.Backend/src/ByteBuy.Core/DTO/Address/ShippingAddressCheckout.cs
namespace ByteBuy.Core.DTO.Address;

public record ShippingAddressCheckout(
    Guid Id,
    string Label,
    string Street,
    string PostalCity,
    string PostalCode,
    string City,
    string HouseNumber,
    string? FlatNumber,
    bool IsDefault
    );

File: Byte&Buy.AngularSPA/src/app/core/dto/shipping-address/shipping-address-checkout.ts
import { Guid } from "guid-typescript";

export interface ShippingAddressCheckout {
    id: Guid;
    label: string;
    street: string;
    postalCity: string;
    postalCode: string;
    city: string;
    houseNumber: string;
    flatNumber?: string;
    isDefault: boolean;
}